tests/test_backends.py

```diff
@@ -149,3 +150,7 @@ def test_superposition_for_all_qubits(connected_backend):
 
 # TODO: test_circuit_result_tensor
 # TODO: test_circuit_result_representation
+
+@pytest.mark.qpu
```

Comment:
I think you don't need to use this marker for this PR. It should be possible to construct platforms and check if they exist without access to instruments. `create_platform` should work without connecting for all instruments.